src/main/java/org/apache/commons/cli/ParseException.java

```diff
@@ -21,11 +21,33 @@ Licensed to the Apache Software Foundation (ASF) under one or more
  * Base for Exceptions thrown during parsing of a command-line.
  */
 public class ParseException extends Exception {
+
     /**
      * This exception {@code serialVersionUID}.
      */
     private static final long serialVersionUID = 9112808380089253192L;
 
+    /**
+     * Converts any exception except {@code UnsupportedOperationException} to a {@code ParseException}.
+     * if {@code e} is an instance of {@code ParseException} it is returned, otherwise a {@code ParseException} is 
+     * created that wraps it.
+     * <p>
```

Comment:
Hello @Claudenw 
- Close HTML tags
- Add `@since` tags to new `public` and `protected` elements.